public/squawk.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+  <link rel="stylesheet" href="squawk.css">
```

Comment:
It's purley to hide the H1. This could be inline, and might not even be necessary, as the iframe should be hidden.